subsys/bluetooth/audio/Kconfig.has

```diff
@@ -0,0 +1,43 @@
+# Bluetooth Audio - Hearing Access Service options
+#
+# Copyright (c) 2022 Codecoup
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_HAS
+	bool "Hearing Access Service support [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  This option enables support for Hearing Access Service.
+
+if BT_HAS
+choice BT_HAS_HEARING_AID_TYPE_CHOICE
+	prompt "Hearing Aid Type selection"
+	help
+	  Select the Hearing Aid Type to compile.
+
+config BT_HAS_HEARING_AID_MONAURAL
+	bool "Monaural Hearing Aid"
+
+config BT_HAS_HEARING_AID_BINAURAL_LEFT
+	bool "Left Binaural Hearing Aid"
+
+config BT_HAS_HEARING_AID_BINAURAL_RIGHT
+	bool "Right Binaural Hearing Aid"
```

Comment:
![image](https://user-images.githubusercontent.com/2520413/158344906-24ebf042-5fa9-4f6c-80b8-8f7fac2fba7c.png)

Should HAS be responsible for `depends on CSIS`, or should HAP introduce this dependency?